Build-Core.cmd

```diff
@@ -2,7 +2,7 @@
 setlocal
 
 SET CMDHOME=%~dp0.
-SET BUILD_FLAGS=/m:1 /v:m /fl /flp:logfile=build.log;verbosity=detailed
```

Comment:
Unless I'm misunderstanding and the CI system consumes the log via a different channel...